### ISSUE TITLE
Deprecate or Disable? Timeline after 15th Oct?

### ISSUE THREAD

**Issue by ianjmiller64** [MicrosoftDocs/microsoft-365-docs#2758]
This doesn't describe the timeline or process for disabling TLS1.0/1.1 for commercial customers after 15th October 2020 - only that deprecation  **begins** 15th October.

Also, while it states for GCC High or DoD clients TLS1.0/1.1 become unsupported, it doesn't explain what the consequences of 'deprecation' are for commercial clients. 

- other support pages say TLS1.2 becomes 'mandatory', but the term 'deprecate' doesn't normally mean _removal_ of a capability - only that it is not advised or supported & will only be finally removed at some future date
 - please clarify consequences of 'deprecation' on commercial clients from 15th and what timeline from 15th Oct onwards is. (e.g. is it USA on 15th, EMEA on 16th, then Asia on 17th for instance?)
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8f7e4139-a562-6b81-6584-277efc4f6103
* Version Independent ID: e812b33a-f183-8b6a-bab0-447ddab2f26f
* Content: [Preparing for TLS 1.2 in Office 365 and Office 365 GCC - Microsoft 365 Compliance](https://docs.microsoft.com/en-us/microsoft-365/compliance/prepare-tls-1.2-in-office-365?view=o365-worldwide)
* Content Source: [microsoft-365/compliance/prepare-tls-1.2-in-office-365.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/prepare-tls-1.2-in-office-365.md)
* Service: **o365-seccomp**
* GitHub Login: @WorkShay
* Microsoft Alias: **shmehta**

**Comment by msbemba** [comment]
@ianjmiller64 Thank you for your feedback.  The deprecation is set to start on October 15 2020. We donot have a date when this will be disabled completely. The consequences are mentioned in the document - https://docs.microsoft.com/en-us/microsoft-365/compliance/tls-1-2-in-office-365-gcc?view=o365-worldwide

If you do not update to TLS version 1.2 (or a later version) by January 15, 2020, you will experience issues when you try to connect to Office 365. Additionally, you will be required to update to TLS 1.2 (or a later version) as part of the resolution.

We have also created a PR on this document to add this information and it can be tracked via#https://github.com/MicrosoftDocs/microsoft-365-docs/pull/2794

Thanks
Sri

**Comment by ianjmiller64** [comment]
thanks Sri, although I was asking about standard O365 - not GCC High or DoD.
I understand from 15th October it will begin for commercial customers of standard O365 and that by 'Deprecate' you actually mean 'Disable'. 
Taking the consequences as being the same for standard version & that you are actually disabling TLS1.0/1.1 we will have to be connecting using TLS1.2 or higher by 15th October 2020 to continue successfully connecting because you can't give us any finer-grained timeline after the 15th October.

Is this correct understanding for O365 standard?.
Thanks, Ian

**Comment by WorkShay** [comment]
What Sri has said is applicable to standard O365 with the exception of start date being Oct 15, not Jan 1.

I think there is some confusion of terms here. So I’ll use the term studio definitions as that’s what we rely on publicly.
End support – “The date when Microsoft no longer provides automatic fixes, updates, or online technical assistance.”
Deprecate – “The stage of the product lifecycle when a feature or functionality is no longer in active development and may be removed in future releases of a product or online service.”
Disable  - “To make an entity nonfunctional.”

Essentially deprecate = end support + disable

Microsoft 365 publicly ended support a few years ago. While we still had TLS 1.0/1.1 calls in our services and systems, if someone had a problem with TLS 1.0/1.1, our feedback was (should’ve been) upgrade to TLS 1.2. The use of the term “deprecate” is purposeful because not all teams will be “disabling” TLS immediately. This is a very similar message, but nuanced as it still allows for the occasional usage of TLS 1.0/1.1 to go through. As I believe you’re aware, internally we have set dates for DoD and GCC-H of Jan1 – July 15. What this means is that all TLS 1.0/1.1 calls in DoD and GCC-H are already unsupported will start getting removed in future releases of products and services within M365 starting Jan. Customers have been told that things will break starting Jan 1 if they use those protocols and to upgrade. In regards to “standard O365”, that date is Oct 15. Customers should have been using TLS 1.2 to connect to O365 since 2017 when we first ended support, but now they have been warned that using TLS 1.0/1.1 in places where it used to work will no longer work. So yes, customers should be connecting using TLS 1.2+ asap because things will start to break by Oct 15. There is no finer-grained timeline.

Best,
Shayna
(they/she)
